app/components/UI/Checkbox.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react'
+import { asField } from 'informed'
+import PropTypes from 'prop-types'
+import { Flex } from 'rebass'
+import styled from 'styled-components'
+
+import Check from 'components/Icon/Check'
+import { Text } from 'components/UI'
+
+const mapOutlineBorderColor = props => {
+  const {
+    disabled,
+    checked,
+    theme: {
+      colors: { gray, lightningOrange }
+    }
+  } = props
+  return !checked || disabled ? gray : lightningOrange
+}
+
+const mapOutlineBackgroundColor = props => {
+  const {
+    disabled,
+    checked,
+    theme: {
+      colors: { lightningOrange }
+    }
+  } = props
+  return !checked || disabled ? 'transparent' : lightningOrange
+}
+
+const CheckboxOutline = styled(Flex)`
+  height: 18px;
+  border-radius: 5px;
+  border: 1px solid ${mapOutlineBorderColor};
+  background-color: ${mapOutlineBackgroundColor};
+`
+
+function mapContainerBorderColor(props) {
+  const {
+    disabled,
+    theme: {
+      colors: { gray, lightningOrange }
+    }
+  } = props
+  return disabled ? gray : lightningOrange
+}
+
+const Container = styled(Flex)`
+  cursor: ${props => (props.disabled ? 'auto' : 'pointer')};
+  user-select: none;
+  &:hover ${CheckboxOutline} {
+    border-color: ${mapContainerBorderColor};
+  }
+`
+
+// Hide checkbox visually but remain accessible to screen readers.
+// Source: https://polished.js.org/docs/#hidevisually
+const HiddenCheckbox = styled.input.attrs({ type: 'checkbox' })`
+  border: 0;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`
+
+const Checkmark = ({ checked, disabled }) => {
+  if (disabled) {
+    return <DisabledCheck />
+  }
+  return checked && <Check width={13} height={13} />
+}
+
+const DisabledCheck = styled.div`
+  width: 6px;
+  height: 0;
+  border: 1px solid gray;
+  border-radius: 5px;
+`
+
+const Checkbox = ({ text, checked, disabled, color, onChange, ...rest }) => (
+  <Container
+    flexDirection="row"
+    alignItems="center"
+    onClick={onChange}
+    {...rest}
+    disabled={disabled}
+    checked={checked}
+  >
+    <HiddenCheckbox checked={checked} onChange={() => {}} />
+    <CheckboxOutline
+      borderRadius={6}
+      width={18}
+      disabled={disabled}
+      color={disabled ? 'gray' : 'primaryColor'}
+      checked={checked}
+      alignItems="center"
+      justifyContent="center"
+    >
+      <Checkmark checked={checked} disabled={disabled} />
+    </CheckboxOutline>
+    <Text ml={2} color="gray">
+      {text}
+    </Text>
+  </Container>
+)
+
+Checkbox.defaultProps = {
+  checked: false,
+  disabled: false
+}
+
+Checkbox.propTypes = {
+  text: PropTypes.string.isRequired,
```

Comment:
The radios have a `label` and a `description` props. The label is the primary colour, whilst the description uses the gray colour. It would good if we could mirror that here.

At a minimum we should use the prop name `label` rather than `text` which would match other inputs, such as the Radios. It would be nice (although I don't think it's needed at this stage) if we could also add a `description`, which would show below the label in the more muted gray colour.

Reply:
Changed text to label. We can also added description later if needed